planning/behavior_velocity_planner/include/scene_module/intersection/scene_intersection.hpp

```diff
@@ -145,6 +145,7 @@ class IntersectionModule : public SceneModuleInterface
   int64_t lane_id_;
   std::string turn_direction_;
   bool has_traffic_light_;
+  bool is_go_out_;
```

Comment:
@rej55 
I thinik this parameter should be in the state machine.

Reply:
I implemented it this way because it is difficult to distinguish between the decision state in autoware and the actual behavior.
So, I think it should be separated from the state machine because the meaning of `is_go_out_` is different from the state in the state machine.

Reply:
if this is RTC specific parameter please name it better like is_activated_after_passing or something.
Because state of GO_OUT is used in traffic light module and this will make confusion

Reply:
Discussion with @rej55 -san 
state_machine : has state if it possible to go or not
is_go_out_ : has state if ego is going out or not
Note: state machine of traffic light module will be changed according to this.